app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
-[ ] please make sure the belongs_to side between Like and User is formated the same as in the Post class